mvpa2/generators/permutation.py

```diff
@@ -58,14 +58,16 @@ def __init__(self, attr, count=1, limit=None, assure=False,
           or sequence thereof) attribute value, where all key-value combinations
           across all given items define a "selection" of to-be-permuted samples
           or features.
-        strategy : 'simple', 'uattrs'
+        strategy : 'simple', 'uattrs', 'chunks'
           'simple' strategy is the straightfoward permutation of attributes (given
           the limit).  In some sense it assumes independence of those samples.
           'uattrs' strategy looks at unique values of attr (or their unique
           combinations in case of `attr` being a list), and "permutes" those
           unique combinations values thus breaking their assignment to the samples
           but preserving any dependencies between samples within the same unique
-          combination.
+          combination. The 'chunks' strategy swaps attribute values of entire chunks.
+          Naturally, this will only work if there is the same number of samples in
+          all chunks.
```

Comment:
I am not exactly sure what you mean, do you mean case for example after hrf modeling, you'll end up with one sample per condition and without order within chunks, so every chink have samples in the same order? In that case swapping labels by that strategy wouldn't do anything